django/utils/timezone.py

```diff
@@ -191,7 +191,7 @@ def now():
     """
     if settings.USE_TZ:
         # timeit shows that datetime.now(tz=utc) is 24% slower
```

Comment:
This is technically an unrelated change, but per [this blog post](https://blog.ganssle.io/articles/2019/11/utcnow.html), I don't think it's a good idea in general to use`.utcnow`. From my tests, `%timeit datetime.now(tz=pytz.UTC)` is approximately as fast as `datetime.utcnow().replace(tzinfo=utc)` now, and when `utc` starts pointing to `datetime.timezone.utc` instead of `pytz.UTC`, `datetime.now(utc)` will be *dramatically* faster, so may as well make that upgrade seamless while we're at it.

If you want me to move this to a separate PR please let me know.

Reply:
Perhaps just a separate commit is fine.

Reply:
My micro-optimizations from 2011 are so cute :-)

https://github.com/django/django/commit/9b1cb755a28f020e27d4268c214b25315d4de42e#diff-698cf55de9163a8fc8381fc3f0320d8336203c45e03306f497c67495bffce40bR224

Then at some point I was the Python 3 guy but I wasn't aware of that subtlety with the datetime module!

Thanks for cleaning that up.

Reply:
Pulled this into #13916